internal/analyzers/govalid/govalid.go

```diff
@@ -178,7 +178,7 @@ func analyzeMarker(pass *codegen.Pass, markersInspect markers.Markers, structTyp
 	return analyzed
 }
 
-func makeValidator(pass *codegen.Pass, markers markers.MarkerSet, field *ast.Field, structName string) []validator.Validator {
+func makeValidator(pass *codegen.Pass, markers markers.MarkerSet, field *ast.Field, structName, parentPath string) []validator.Validator {
```

Comment:
At some point, it should be considered to pass a struct as argument here instead of adding argument over and over.

Maybe it's the moment